### PR TITLE
Agnoplay integration docs complete.

### PR DESCRIPTION
**Pull request by bengo1990**
Docs complete.

### CODE REVIEW

**Review by fschurer — APPROVED** [review]

File: docs/source/DeveloperDocumentation/PlayerIntegrations/agnoplay.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Agnoplay
+
+[Agnoplay](https://www.agnoplay.com/) is a fully agnostic, cloudbased player service.
+
+The [agnoplay](https://github.com/Vualto/agnoplay) repository demonstrates at a lower level how to integrate [VUDRM](https://docs.vualto.com/projects/vudrm/en/latest/index.html) with Agnoplay.
+
+## Example of Agnoplay Integration
+
+This files below can be found in the repo mentioned above here:
```
Comment (fschurer) [inline docs/source/DeveloperDocumentation/PlayerIntegrations/agnoplay.md:9]: Typo (this files should be the files)

**Review by fschurer — DISMISSED** [review]

File: docs/source/DeveloperDocumentation/PlayerIntegrations/agnoplay.md (hunk 1 above)
Comment (bengo1990) [reply docs/source/DeveloperDocumentation/PlayerIntegrations/agnoplay.md:9]: The sentence before made my brain melt, changed!

**Review by somerobmitchell — APPROVED** [review]

**Comment by somerobmitchell** [comment]
blocked until https://github.com/Vualto/agnoplay/pull/1 is complete

**Review by fschurer — APPROVED** [review]

File: docs/source/DeveloperDocumentation/PlayerIntegrations/agnoplay.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                }
+            },
+            playready: {
+                licenseUrl: "https://playready-license.vudrm.tech/rightsmanager.asmx",
+                licenseHeaders: {
+                    "x-vudrm-token": vudrmToken
+                }
+            },
+            fairplay: {
+                licenseUrl: "https://fairplay-license.vudrm.tech/v2/license/",
```
Comment (somerobmitchell) [inline docs/source/DeveloperDocumentation/PlayerIntegrations/agnoplay.md:79]: @bengo1990 is there no way to extract the fairplay license URL from the HLS manifest?
This is the preferred approach. If it is not possible then the content id will need to either be appended to the URL _or_ included in the VUDRM token.

File: docs/source/DeveloperDocumentation/PlayerIntegrations/agnoplay.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                licenseHeaders: {
+                    "x-vudrm-token": vudrmToken
+                },
+                certificateUrl: "https://fairplay-license.vudrm.tech/certificate/",
```
Comment (somerobmitchell) [inline docs/source/DeveloperDocumentation/PlayerIntegrations/agnoplay.md:83]: You no longer need to attach the VUDRM token as a header to the cert request. The preferred way of doing this now is to not use a VUDRM token and to set the client name in the URL:

`https://fairplay-license.vudrm.tech/certificate/<client-name>

**Review by somerobmitchell — CHANGES_REQUESTED** [review]

**Pull request closed without merging**